# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cmake/hotfix.patch.uclibc
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Source/kwsys/testDynamicLoader.cxx.vanilla	2022-11-02 20:02:12.527095156 +0100
+++ ./Source/kwsys/testDynamicLoader.cxx	2022-11-02 20:05:40.623094243 +0100
@@ -13,13 +13,15 @@
 #  include <features.h>
 
 // Will define LIBDL_SO macro on systems with glibc.
+#  ifndef __UCLIBC__
 #  ifdef __GLIBC__
-#    include <gnu/lib-names.h>
+#    include <gnu/lib-names.h>
 // Define to LIBC_SO, if not defined by above header.
 #    ifndef LIBDL_SO
 #      define LIBDL_SO LIBC_SO
 #    endif
 #  endif
+#  endif
 
 // Define the LIBDL_SO macro, if not defined above.
 #  ifndef LIBDL_SO
